docs/section6/Rule6-4.md

```diff
@@ -0,0 +1,30 @@
+
+# Lighting - Rule 6-4
+
+**Rule ID:** 6-4  
+**Rule Description:** Where a complete lighting system exists, the actual lighting power for each thermal block shall be used in the model.  
+**Appendix G Section:** Section G3.1-6(a) Modeling Requirements for the Proposed Design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+## Rule Logic: 
+
+- For each building_segment in the proposed model: ```building_segment_p in P_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```thermal_block_p in building_segment_p.thermal_blocks:```  
+
+    - For each zone in thermal block: ```zone_p in thermal_block_p.zones:```  
+
+      - For each space in zone: ```space_p in zone_p.spaces:```  
+
+        - Get interior lighting in U_RMR: ```interior_lighting_b = match_data_element(U_RMR, InteriorLightings, space_p.interior_lighting.id)```
+
+          **Rule Assertion:** 
+
+          - Case 1: Lighting power in the proposed RMR is as existing and cannot be verified by RCT: ```if space_p.interior_lighting == interior_lighting_b: CAUTION```
+
```

Comment:
I don't think this assertion works because there are no attributes in the RMR that allow differentiating between projects with existing lighting vs newly designed lighting vs lighting that is not designed. So all steps prior to row 28 would be identical for these three cases. So then how can RCT say that "Lighting power in the proposed RMR is as existing and cannot be verified by RCT"? I think in all three cases it should say "Lighting power in the proposed RMR is the same as in the baseline RMR". Then RCT can throw either "Pass" or "Caution" flag.

Reply:
Updated with manual check and new rule assertion. @mkarpman @supriyagoel @JasonGlazer 
Same question here on if we want all fields in InteriorLighting be the same between U_RMR and P_RMR or just power_per_area, lighting_multiplier_schedule, has_occupancy_control, has_daylighting_control.